docs/source/receptorctl/receptorctl_work_submit.rst

```diff
@@ -0,0 +1,39 @@
+-----------
+work submit
+-----------
+
+.. contents::
+   :local:
+
+``receptorctl work submit`` submits a new unit of work.
```

Comment:
Maybe it's a bit more wordy but I never like using the same word as the command option to describe what happens. What about?

"requests a Receptor node to run a unit of work."?